src/utils.ts

```diff
@@ -133,3 +133,18 @@ export const sanitiseFilename = (filename: string) => {
     .replace(windowsReservedRe, '_')
     .toLocaleLowerCase();
 };
+
+export const getApiName = () =>
+  fs.readdirSync(join(__dirname, '../airkeeper-deployment', 'templates', getVersion())).pop();
```

Comment:
Should we check for `undefined` before `pop()`?

Reply:
I was thinking about that, it's an unlikely/perhaps impossible scenario under normal conditions, but worth having a check regardless, thank you :)
There are a number of areas in this script/related components where things could go wrong if the structure of the repository isn't as expected.

Reply:
Then maybe it will be more productive to just add all those checks in a separate PR